specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -6800,6 +6812,12 @@
           "readOnly": true,
           "type": "string",
           "description": "The provisioning state."
+        },
+        "hyperVGeneration": {
+          "hyperVGenerationType": {
```

Comment:
@vamuzumd that is not valid swagger. It should be:
``` json
"hyperVGeneration" : {
    "$ref": "#/definitions/HyperVGenerationType",
    "description": "Gets the HyperVGenerationType of the image"
}
```

Reply:
yes, I will make the change